src/lamplib/src/genny/toolchain.py

```diff
@@ -184,13 +184,18 @@ def toolchain_info(
 
 class ToolchainDownloader(Downloader):
     # These build IDs are from the genny-toolchain Evergreen task.
-    # https://evergreen.mongodb.com/waterfall/genny-toolchain
-    # Find a compile task (for any build variant) and modify the URL:
-    # genny_toolchain_archlinux_t_compile_82eb7c32ad09726f3ef0ddc8d7f24a18b03d9644_21_11_23_16_37_21
-    # =>                                  82eb7c32ad09726f3ef0ddc8d7f24a18b03d9644_21_11_23_16_37_21
+    # Old UI: https://evergreen.mongodb.com/waterfall/genny-toolchain
+    # New UI: https://spruce.mongodb.com/commits/genny-toolchain
+    #
+    # Find a compile task (for any build variant) and modify the URL. 
+    # For example:
+    # https://spruce.mongodb.com/task/genny_toolchain_archlinux_t_compile_82eb7c32ad09726f3ef0ddc8d7f24a18b03d9644_21_11_23_16_37_21
+    # =>                                                                  82eb7c32ad09726f3ef0ddc8d7f24a18b03d9644_21_11_23_16_37_21
+    # https://spruce.mongodb.com/task/genny_toolchain_macos_1100_arm64_t_compile_patch_87457e6fec1d98f270c84d915f83bec53554ecee_6451d23fc9ec4441c9ce233d_23_05_03_03_17_20
+    # =>                                                                         patch_87457e6fec1d98f270c84d915f83bec53554ecee_6451d23fc9ec4441c9ce233d_23_05_03_03_17_20
```

Comment:
[potentially bike-shedding so this is being left as a comment. Given that John has approved this, this feedback is not blocking!]

The original comment is describing how this function works normally. The added comment (I think) is introducing a new URL scheme (`patch_*`) without explaining that this is the URL scheme for a PR build or why using that would be useful. I am not sure this is adding clarity.

The URLs are appreciated. Could you please modify the comment to indicate that `82eb7c32ad09726f3ef0ddc8d7f24a18b03d9644_21_11_23_16_37_21` is the normal scheme, and `patch_87457e6fec1d98f270c84d915f83bec53554ecee_6451d23fc9ec4441c9ce233d_23_05_03_03_17_20` is what a patch build will look like? Perhaps having multiple forms of ID in the comment can be made clearer by introducing `https://spruce.mongodb.com/task/genny_toolchain_<$DISTRO>_t_compile_<$ID>` and giving examples of $DISTRO and $ID.

Reply:
Yes I've added these details, thank you!